src/main/java/de/tum/in/www1/artemis/service/compass/umlmodel/classdiagram/UMLClassDiagram.java

```diff
@@ -109,7 +109,7 @@ public boolean isUnassessed() {
      *
      * @return isEntirelyAssessed
      */
-    // TODO CZ: remove as it is not needed anymore?
+    @SuppressWarnings("unused")
```

Comment:
I could also remove this method as it is not used anymore. I thought we could keep it for now as we might need it in the future.